EngineLayer/CrosslinkSearch/CrosslinkSearchEngine.cs

```diff
@@ -20,6 +20,7 @@ public class CrosslinkSearchEngine : ModernSearchEngine
 
         private readonly bool CrosslinkSearchTopN;
         private readonly int TopN;
+        private readonly bool NoCleaveAtCrosslinkSite;
```

Comment:
This is a really petty thing, but a name like "CleaveAtCrosslinkSite" is a lot cleaner than "NoCleaveAtCrosslinkSite". I understand this would require you to invert the the conditionals, but it's just kind of a weird as is.

Reply:
Agreed.